### PR TITLE
more changes for the routes

### PR DESCRIPTION
**Pull request by unclejack**
Makes changes needed for #229.

### CODE REVIEW

File: volmaster/daemon.go [hunk 1]
```diff
@@ -90,15 +90,26 @@ func (d *DaemonConfig) Daemon(listen string) {
 		r.HandleFunc(path, logHandler(path, d.Global.Debug, f)).Methods("POST")
 	}
 
+	deleteRouter := map[string]func(http.ResponseWriter, *http.Request){
+		"/volumes/remove":           d.handleRemove,
```
Comment (mapuri) [inline volmaster/daemon.go]: @unclejack we should remove these routes from POST router above, right?

**Comment by mapuri** [comment]
LGTM, except for one minor clarification.

File: volmaster/daemon.go (hunk 1 above)
Comment (unclejack) [inline volmaster/daemon.go]: @mapuri You're right. I'll update right away.

File: volcli/volcli.go [hunk 2]
```diff
@@ -56,6 +56,15 @@ func ppJSON(v interface{}) ([]byte, error) {
 	return json.MarshalIndent(v, "", "  ")
 }
 
+func deleteRequest(url string, bodyType string, body io.Reader) (resp *http.Response, err error) {
```
Comment (erikh) [inline volcli/volcli.go]: why are we passing bodyType without using it?

File: volmaster/daemon.go [hunk 3]
```diff
@@ -220,6 +228,40 @@ func (d *DaemonConfig) handlePolicy(w http.ResponseWriter, r *http.Request) {
...
+
+	_, err := d.Config.GetVolume(policy, volumeName)
+	if config.NotFound(err) {
+		w.WriteHeader(404)
+		return
+	} else if err != nil {
+		httpError(w, "Retrieving volume", err)
+	}
+
+	if err := d.Config.GetUse(mount, vc); err != nil {
```
Comment (erikh) [inline volmaster/daemon.go]: Is the volume fetch necessary here? I'm not sure it is.

**Comment by erikh** [comment]
looks good otherwise

File: volcli/volcli.go (hunk 2 above)
Comment (unclejack) [inline volcli/volcli.go]: I'll fix it.

**Comment by unclejack** [comment]
@erikh The fixes are in and the CI is green. PTAL

File: volcli/volcli.go (hunk 2 above)
Comment (erikh) [inline volcli/volcli.go]: this is still needed

**Comment by erikh** [comment]
looks like we have one last issue here

File: volcli/volcli.go (hunk 2 above)
Comment (unclejack) [inline volcli/volcli.go]: @erikh It's used below to set "Content-Type". That's the fix I've made.

**Comment by erikh** [comment]
aha! LGTM

**Pull request merged**